README.md

```diff
@@ -232,6 +243,20 @@ model = models.get("model-name", pretrained_weights="pretrained-model-name")
  </br></br>
 
 
+### Pose Estimation
+
+#### Transfer Learning
+
+<table class="tfo-notebook-buttons" align="left">
+    <td width="500">   
+        <a target="_blank" href="https://colab.research.google.com/drive/1NMGzx8NdycIZqnRlZKJZrIOqyj0MFzJE#scrollTo=3UZJqTehg0On">
+            <img src="./documentation/assets/SG_img/colab_logo.png" /> Pose Estimation Transfer Learning
+        </a>
+    </td>
+</table>
+</br></br>
+
+https://colab.research.google.com/drive/1NMGzx8NdycIZqnRlZKJZrIOqyj0MFzJE#scrollTo=3UZJqTehg0On
```

Comment:
I think this is not wanted

Reply:
Why? What is wanted instead?

Reply:
Fixed in dd3efe7de4b591f293b528f9a3ed1ad72ebcfbf2